.github/workflows/terraform.yml

```diff
@@ -0,0 +1,88 @@
+# This workflow will provision the infrastructure necessary to deploy our application.
+
+name: terraform
+
+on:
+    push:
+        branches: [ main ]
+        paths:
+            - 'terraform/*'
+            - '.github/workflows/terraform.yaml'
+    pull_request:
+        branches: [ main ]
+        paths:
+            - 'terraform/*'
+            - '.github/workflows/terraform.yaml'
+jobs:
+  format:
+    permissions:
+      contents: write
+    runs-on: ubuntu-latest
+
+    name: Format
+    
+    steps:
+      # Clone the repository.
+      - uses: actions/checkout@v2
```

Comment:
There is a much later version of this action available. Why is this version being use? Copy / Paste coding?